tests/cupyx_tests/scipy_tests/fft_tests/test_fft.py

```diff
@@ -1627,6 +1627,176 @@ def test_ihfft_backend(self, xp, dtype):
         return _correct_np_dtype(xp, dtype, out)
 
 
+@testing.parameterize(*(
+    testing.product_dict([
+        {'shape': (2, 5), 's': None, 'axes': None},
+        {'shape': (2, 10), 's': (1, 5), 'axes': None},
+        {'shape': (2, 30), 's': None, 'axes': (-2, -1)},
+        {'shape': (2, 50), 's': None, 'axes': (-1, -2)},
+        {'shape': (2, 100), 's': (2, 50), 'axes': (0,)}
+    ],
+        testing.product({'norm': [None, 'backward', 'ortho', 'forward']})
+    )
+))
+@testing.gpu
+@testing.with_requires('scipy>=1.4.0')
+class TestHfft2(unittest.TestCase):
+
+    def setUp(self):
+        _skip_forward_backward(self.norm)
+
+    @testing.for_all_dtypes()
+    @testing.numpy_cupy_allclose(rtol=4e-4, atol=1e-7, accept_error=ValueError,
+                                 contiguous_check=False)
+    def test_hfft2(self, xp, dtype):
+        x = testing.shaped_random(self.shape, xp, dtype)
+        x_orig = x.copy()
+        out = _fft_module(xp).hfft2(x, s=self.s, axes=self.axes,
+                                    norm=self.norm)
+        testing.assert_array_equal(x, x_orig)
+        return _correct_np_dtype(xp, dtype, out)
+
+    @testing.for_all_dtypes()
+    @testing.numpy_cupy_allclose(rtol=4e-4, atol=1e-7, accept_error=ValueError,
+                                 contiguous_check=False)
+    def test_hfft2_overwrite(self, xp, dtype):
+        x = testing.shaped_random(self.shape, xp, dtype)
+        overwrite_kw = {} if xp is np else {'overwrite_x': True}
+        out = _fft_module(xp).hfft2(x, s=self.s, axes=self.axes,
+                                    norm=self.norm, **overwrite_kw)
+        return _correct_np_dtype(xp, dtype, out)
```

Comment:
This test is currently no different from the above one, as `overwrite_x` is not yet supported. IMHO it can be either kept as is or commented out.

Reply:
Shall I remove this test? We could add it again once `overwrite_x` is functional

Reply:
yeah let's take them out